viewing_party/party.py

```diff
@@ -1,23 +1,136 @@
 # ------------- WAVE 1 --------------------
 
+from statistics import StatisticsError, mode
+
+
 def create_movie(title, genre, rating):
-    pass
+    if bool(title) == True and bool(genre) == True and bool(rating) == True:

```

Comment:
Cool way to check for truthy-ness! A more "pythonic" way to express conditionals for boolean variables is:
`if bool(title) and bool(genre) and bool(rating):`
It can be further simplified:
`if title and genre and rating:`